### PR TITLE
Fill in content to match website mockup

### PR DESCRIPTION
**Pull request by waxlamp**

### CODE REVIEW

**Comment by waxlamp** [comment]
@opadron please take a look.

**Comment by opadron** [comment]
Looks good after a quick first pass.  I'm going to merge right away and have a look at the github page.

File: index.html [hunk 1]
```diff
@@ -129,295 +123,129 @@ <h4 class="service-heading">Web Security</h4>
...
-                            </li>
-                            <li><a href="#"><i class="fa fa-linkedin"></i></a>
-                            </li>
+                        <img src="img/people/haoding.png" class="img-responsive img-circle" alt="">
+                        <h4>Hao Ding</h4>
+                        <p class="text-muted">Ph.D. Candidate in Computer Science &amp; Engineering</p>
+                        <ul class="list-unstyled">
+                            <li>The Ohio State University</li>
+                            <li><i>tel:</i> 614-xxx-xxxx</li>
+                            <li><i>email</i>: ding.96@osu.edu</li>
```
Comment (opadron) [inline index.html:186]: Let's make these `mailto:` links.

File: index.html [hunk 2]
```diff
@@ -129,295 +123,129 @@ <h4 class="service-heading">Web Security</h4>
...
-                            </li>
-                            <li><a href="#"><i class="fa fa-linkedin"></i></a>
-                            </li>
+                        <img src="img/people/kunhuang.png" class="img-responsive img-circle" alt="">
+                        <h4>Kun Huang, Ph.D.</h4>
+                        <p class="text-muted">Associate Professor, College of Medicine</p>
+                        <ul class="list-unstyled">
+                            <li>The Ohio State University</li>
+                            <li><i>tel:</i> 614-292-5607</li>
+                            <li><i>email</i>: kun.huang@osumc.edu</li>
```
Comment (opadron) [inline index.html:198]: Let's make these `mailto:` links.

File: index.html [hunk 3]
```diff
@@ -129,295 +123,129 @@ <h4 class="service-heading">Web Security</h4>
...
-                            </li>
-                            <li><a href="#"><i class="fa fa-linkedin"></i></a>
-                            </li>
+                        <img src="img/people/raghumachiraju.png" class="img-responsive img-circle" alt="">
+                        <h4>Raghu Machiraju, Ph.D.</h4>
+                        <p class="text-muted">Professor, College of Engineering and College of Medicine</p>
+                        <ul class="list-unstyled">
+                            <li>The Ohio State University</li>
+                            <li><i>tel:</i> 614-2926730</li>
+                            <li><i>email</i>: imachiraju.1@osu.edu</li>
```
Comment (opadron) [inline index.html:210]: Let's make these `mailto:` links.

**Comment by opadron** [comment]
Just a minor tweak about the email links, otherwise it's fine for now.  If you update the links and send another PR in, feel free to merge it immediately.

**Pull request merged**